DQM/SiStripCommon/test/plugins/testTkHistoMap.cc

```diff
@@ -48,13 +50,18 @@ class testTkHistoMap : public DQMOneEDAnalyzer<> {
   const edm::ESGetToken<TkDetMap, TrackerTopologyRcd> tTopoToken;
   const edm::ESGetToken<TrackerGeometry, TrackerDigiGeometryRecord> tkGeoToken;
 
-  bool readFromFile;
+  const bool readFromFile_;
+  const edm::FileInPath fp_;
   std::unique_ptr<TkHistoMap> tkhistoID, tkhisto, tkhistoBis, tkhistoZ, tkhistoPhi, tkhistoR, tkhistoCheck;
 };
 
 //
 testTkHistoMap::testTkHistoMap(const edm::ParameterSet& iConfig)
-    : tTopoToken(esConsumes()), tkGeoToken(esConsumes()), readFromFile(iConfig.getParameter<bool>("readFromFile")) {}
+    : tTopoToken(esConsumes()),
+      tkGeoToken(esConsumes()),
+      readFromFile_(iConfig.getParameter<bool>("readFromFile")),
+      fp_(iConfig.getUntrackedParameter<edm::FileInPath>("inputFile",
```

Comment:
This should probably be a tracked parameter as the `readFromFile` is tracked. You can give defaults for tracked parameter if you use a `fillDescription` function.